bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -26,6 +26,16 @@
 
 # import libROOTPythonizations with Python version number
 import sys, importlib
+import warnings
+
+if sys.version_info.major < 3:
+    # DeprecationWarning is ignored by default in Python, the next line enables it
+    warnings.filterwarnings("default", category=DeprecationWarning,
+                            module=__name__)
+    warnings.warn(
```

Comment:
This can cause havoc with experiments' tests. Should we have this only during configure time or is there a strong reason to keep it here? Otoh maybe all experiments moved away from Python 2, one would hope so...

Reply:
I'm not sure tbh, the important bit of the PR is turning OFF building Python 2 support by default. If you fear this extra warning may be too aggressive, I will remove it. This warning can only happen if ROOT was built with `pyroot-python2` option, which means that experiments' stack need to be adapted anyway if they still want to run with Python 2 support

Reply:
In the end I think I'm going to leave the warning there, if there is some stack still relying on Python 2, they should make the move too. Let's see how much noise this will cause